cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumConstants.java

```diff
@@ -21,6 +22,8 @@
     public static final String STRIMZI_DOMAIN = "strimzi.io/";
     public static final String STRIMZI_IO_USE_CONNECTOR_RESOURCES = STRIMZI_DOMAIN + "use-connector-resources";
 
+    public static final LocalObjectReference IMAGE_PULL_SECRET = new LocalObjectReference("cos-pull-secret");
```

Comment:
@ani-sha can you file a new Jira task for making it configurable via a Secret? We can add this config option later because Hiram is waiting eagerly to have this live on staging and tomorrow will be release train departure.
 
btw @lburgazzoli is there really a need to have a secret to configure the name of another secret? Can't we just agree on this global secret name?

Reply:
it does need to be a secret, it can be an env var set by the bundle (and we can have `cos-pull-secret` as default value).